assets/webconfig/js/content_general.js

```diff
@@ -28,6 +28,7 @@ $(document).ready( function() {
 	// Token handling
 	function buildTokenList()
 	{
+		console.log(tokenList)
```

Comment:
Could you please delete this line?
Thank you!

Reply:
Yes. no Problem